packages/openlmis/src/Adaptor.js

```diff
@@ -0,0 +1,132 @@
+import { expandReferences } from '@openfn/language-common/util';
+import { execute as commonExecute } from '@openfn/language-common';
+import * as util from './Utils';
+
+/**
+ * State object
+ * @typedef {Object} OpenLMISState
+ * @private
+ * @property data - the parsed response body
+ * @property response - the response from the HTTP server, including headers, statusCode, body, etc
+ * @property references - an array of all previous data objects used in the Job
+ **/
+
+/**
+ * Options provided to the HTTP request
+ * @typedef {Object} RequestOptions
+ * @property {object} body - body data to append to the request. JSON will be converted to a string (but a content-type header will not be attached to the request).
+ * @property {object} query - An object of query parameters to be encoded into the URL.
+ * @property {object} headers - An object of headers to append to the request.
+ */
+
+/**
+ * Execute a sequence of operations.
+ * Wraps `language-common/execute`, and prepends initial state for odk.
+ * @example <caption> * execute(</caption>
+ *   get("processingPeriods")
+ * )(state)
+ * @private
+ * @param {Operations} operations - Operations to be performed.
+ * @returns {Operation}
+ * @state {OpenLMISState}
+ */
+export function execute(...operations) {
+  return state => {
+    return commonExecute(util.authorize, ...operations)(state);
+  };
+}
+/**
+ * Make a GET request in OpenLMIS
+ * @example <caption>Get all supplyLines</caption>
+ * get("supplyLines");
```

Comment:
I'd love this example to look like a path - can we at least do `get("/supplyLines");` ?

The same for the other requests and examples

Reply:
I have updated the examples `path`